x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/IdentifierGenerator.java

```diff
@@ -40,22 +48,27 @@ public static String randomIndexPattern(Feature... features) {
             index.append('.');
         }
         index.append(randomCharacterFrom(validFirstCharacters));
-        for (int i = 0; i < ESTestCase.randomIntBetween(1, 100); i++) {
+        for (int i = 0; i < randomIntBetween(1, 100); i++) {
             index.append(randomCharacterFrom(validCharacters));
         }
         if (canAdd(Features.WILDCARD_PATTERN, features)) {
```

Comment:
Also out of scope, but I still think we should have patterns with more than 1 wildcard as 1 wildcard character is something I can see us accidentally special case for.

Reply:
Added